__tests__/app.test.js

```diff
@@ -36,14 +31,54 @@ describe("/api", () => {
       });
     });
   });
+  describe("/reviews", () => {
+    describe("/review_id", () => {
+      test("Should return a review object relating to the passed review_id", () => {
+        return request(app)
+          .get("/api/reviews/1")
+          .expect(200)
+          .then(({ body }) => {
+            const { review } = body;
+            expect(review).toEqual({
+              title: "Agricola",
+              designer: "Uwe Rosenberg",
+              owner: "mallionaire",
+              review_img_url:
+                "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+              review_body: "Farmyard fun!",
+              category: "euro game",
+              created_at: "2021-01-18T10:00:20.514Z",
+              votes: 1,
+              review_id: 1,
+            });
+          });
+      });
+    });
+  });
   describe("Error Handling", () => {
-    test("404: Should return 404 code and handle error when passed an invalid route", () => {
+    test("404: Should return when passed an invalid route", () => {
       return request(app)
         .get("/api/invalidRoute")
         .expect(404)
         .then(({ body }) => {
           expect(body.msg).toEqual("That route does not exist");
         });
     });
+    test("400: Should return when passed an id that is not valid", () => {
+      return request(app)
+        .get("/api/reviews/InvalidId")
+        .expect(404)
```

Comment:
Expecting a 404 instead of a 400?

Reply:
oooops lol, got them the wrong way round initialy and forgot to change it.